pavelib/assets.py

```diff
@@ -378,7 +378,7 @@ def collect_assets(systems, settings):
     `settings` is the Django settings module to use.
     """
     for sys in systems:
-        sh(django_cmd(sys, settings, "collectstatic --noinput > /dev/null"))
+        sh(django_cmd(sys, settings, "collectstatic --clear --noinput -v 0"))
```

Comment:
@jibsheet I think we want to keep the piping to /dev/null for now. The output is fairly voluminous, and Ned complained about it in the dev channel. I have a FedX story to come up with a solution so that developers don't see the output but that we do see it in builds:

https://openedx.atlassian.net/browse/FEDX-152